test/functional/suite_test.go

```diff
@@ -53,17 +54,20 @@ func TestAPIs(t *testing.T) {
 		Groups:   []string{"system:authenticated"},
 	}
 
-	// get RMI CR and if cluster storage set to true, skip the suite
-	context, err := common.NewTestingContext(cfg)
-	if err != nil {
-		t.Fatalf("\"failed to create testing context: %s", err)
-	}
-	rhmi, err := common.GetRHMI(context.Client, true)
-	if err != nil {
-		t.Fatalf("error getting RHMI CR: %v", err)
-	}
-	if rhmi.Spec.UseClusterStorage == "true" {
-		t.Skip("Aborting functional tests: \"UseClusterStorage\" is set to true. \nPlease, run another testing suite or reinstall operator with \"UseClusterStorage\" set to false")
+	// Allow overriding via environment variable
+	if os.Getenv("BYPASS_STORAGE_TYPE_CHECK") != "true" {
```

Comment:
Addong a `BYPASS_STORAGE_TYPE_CHECK` envrironement variable to bypass running functional tests with `useCllusterStorage: true`. Useful for local development where cluster storage is usually used and for tests that do not require external resources :thinking:

Reply:
+1
This env var will definitely be useful going forward.